File: assets/js/areaofTriangle.js
// Problems: Area of a triangle, where the value of base and height are given by the user.

/** Area of Rectangle formula 
 *  A = (base * height) / 2;
*/

let base = +prompt('Enter your base value: '),
    height = +prompt('Enter your height value: ');

// A = (base * height) / 2;
const A = (base * height) / 2;

// Display the result
console.log(`Base: ${base}`);
console.log(`Height: ${height}`);
console.log(`Are of triangle: ${A}`);

File: assets/c/area-of-triangle.c
// Problems: Area of a triangle, where the value of base and height are given by the user.

/** Area of Rectangle formula 
 *  A = (base * height) / 2;
*/

#include <stdio.h>

int main() {
  float b, h, A;

  printf("Enter your base value: ");
  scanf("%f", &b);
  printf("Enter your height value: ");
  scanf("%f", &h);

  // Area of triangle
  A = .5 * b * h;

  // Display the area of triangle
  printf("Area of triangle = %2f", A);

  return 0;
}